daml-lf/spec/value.rst

```diff
@@ -150,35 +150,39 @@ Support for value versions 1 to 5 was dropped on 2020-12-10.
 This breaking change does not impact ledgers created with SDK 1.0.0 or
 later.
 
-
 +--------------------+-----------------+
 | Version identifier | Date introduced |
 +====================+=================+
 +--------------------+-----------------+
-|                  6 |      2019-11-07 |
+|                 10 |      2019-11-07 |
 +--------------------+-----------------+
-|                11  |      2021-01-19 |
+|                 11 |      2021-01-19 |
 +--------------------+-----------------+
 |                dev |      2019-12-14 |
 +--------------------+-----------------+
 
 message VersionedValue
 ^^^^^^^^^^^^^^^^^^^^^^
 
-*since version 1*
-
 The outermost entry point, a `message Value`_ with version annotation.
 
+(*since version 10*)
+
 In this version, these fields are included:
 
 * ``string`` version
 * `message Value`_ value
 
 ``version`` is required, and must be a version of this specification.
-For example, for version 1 of this specification, ``version`` must be
-``"1"``.  Consumers can expect this field to be present and to
-have the semantics defined here without knowing the version of this
-value in advance.
+For backward compatibility reasons:
+- the version `10` is encoded as the string "6";
+- string "10" is reserved and will be never used to encoded any future version;
+- versions 11 or latter will be encoded as string, for instance
+  version 11 of this specification, ``version`` must be ``"1"``.
```

Comment:
```suggestion
  version 11 of this specification, ``version`` must be ``"11"``.
```
or am I missing something? It looks like otherwise we’ll get a collision at 16.